topologies/topology_test/otg_test/topology_test.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2022 Google LLC
```

Comment:
Move to new path?

topologies/otg_tests/topology_test/topology_test.go

Reply:
The original topologies/topology_test/topology_test.go was built for ate (IxNetwork) so I thought I could just put the OTG version inside of that otg_test folder. We should probably create topologies/topology_test/ate_test/ folder and move topology_test.go there.

Reply:
I will update the path on both.